src/applications/gi/components/vet-tec/VetTecHeadingSummary.jsx

```diff
@@ -0,0 +1,115 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import _ from 'lodash';
+
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import VetTecAdditionalResources from '../content/VetTecAdditionalResources';
+import { formatNumber } from '../../utils/helpers';
+
+const IconWithInfo = ({ icon, children, present }) => {
```

Comment:
`show` is probably more idiomatic than `present.` In general we prefer the parent component to handle this behavior e.g. 

recommended: 
```javascript
{ it.website && <IconWithInfo icon="globe"> } 
```

not recommended: 
```javascript
<IconWithInfo icon="globe" present={it.website}>
```